client/jetpack-cloud/sections/partner-portal/assign-license-form/style.scss

```diff
@@ -62,8 +62,14 @@ p.assign-license-form__description {
 	margin-bottom: 1px;
 }
 
-.assign-license-form__site-card-radio.form-radio {
+.assign-license-form__site-card .partner-portal-checkbox {
 	margin-right: 1rem;
+	position: relative;
+	top: 3px;
+}
```

Comment:
Would we be able to skip component specific position styling like this if we inject the label into the radio component and then format it with flex or something?